dashboard/app/assets/javascripts/header.js.erb

```diff
@@ -91,6 +93,26 @@ dashboard.buildHeader = function (stageData, progressData, currentLevelId) {
   $('.header_popup').click(function (e) {
     e.stopPropagation(); // Clicks inside the popup shouldn't close it
   });
+  $('.header_popup_close').click(hideHeaderPopup);
+
+  $(window).resize(function () {
+    if ($('.header_popup').is(':visible')) {
```

Comment:
I've used this before:

  window.addEventListener("resize", function()
  {
    clearTimeout(resizeId);
    resizeId = setTimeout(sizeCanvas, 500);
  });